covid19_sfbayarea/data/marin_scraper.py

```diff
@@ -0,0 +1,303 @@
+		# Each new row has data for a different age group.
+		c_age["group"] = row.split(',')[0]
+		if c_age["group"] not in ages:
+			raise ValueError('The age groups have changed.')
+		c_age["raw_count"] = int(row.split(',')[2])
+		d_age["group"] = row.split(',')[0]
+		d_age["raw_count"] = int(row.split(',')[4])
+		c_brkdown.append(c_age)
+		d_brkdown.append(d_age)
+
+	return c_brkdown, d_brkdown
+
+def get_breakdown_gender(chart_id: str, url: str) -> Tuple:
+	"""This method gets the breakdown of cases and deaths by gender."""
+	csv_ = extract_csvs(chart_id, url)
+
+	csv_strs = csv_.splitlines()
+	keys = csv_strs[0].split(',') 
+	if keys != ['Gender', 'POPULATION', 'Cases', 'Hospitalizations', 'Deaths']:
+		raise ValueError('The headers have changed.')
+
+	genders = ['male', 'female']
+	c_gender = {}
+	d_gender = {}
+	
+	for row in csv_strs[1:]:
+		# Extracting the gender and the raw count (the 3rd and 5th columns, respectively) for both cases and deaths.
+		# Each new row has data for a different gender.
+		split = row.split(',')
+		gender = split[0].lower()
+		if gender not in genders:
+			return ValueError('The genders have changed.')
+		c_gender[gender] = int(split[2])
+		d_gender[gender] = int(split[4])			
+
+	return c_gender, d_gender
+
+def get_breakdown_race_eth(chart_id: str, url: str) -> Tuple:
+	"""This method gets the breakdown of cases and deaths by race/ethnicity."""
+
+	csv_ = extract_csvs(chart_id, url)
+
+	csv_strs = csv_.splitlines()
+	keys = csv_strs[0].split(',') 
+	
+	if keys != ['Race/Ethnicity', 'COUNTY POPULATION', 'Case Count', 'Percent of Cases', 'Hospitalization Count', 'Percent of Hospitalizations', 'Death Count', 'Percent of Deaths']:
+		raise ValueError("The headers have changed.")
+
+	key_mapping = {"black/african american":"African_Amer", "hispanic/latino": "Latinx_or_Hispanic",
+            "american indian/alaska native": "Native_Amer", "native hawaiian/pacific islander": "Pacific_Islander", "white": "White", "asian": "Asian", "multi or other race": "Multi or Other Race"}
+            # "Multiple_Race", "Other" are not separate in this data set - they are one value under "Multi or Other Race"
+
+	c_race_eth = {}
+	d_race_eth = {}
+	
+	for row in csv_strs[1:]:
+		split = row.split(',')
+		race_eth = split[0].lower()
+		if race_eth not in key_mapping:
+			raise ValueError("The race_eth groups have changed.")
+		else:
+			c_race_eth[key_mapping[race_eth]] = int(split[2])
+			d_race_eth[key_mapping[race_eth]] = int(split[6])
+
+	return c_race_eth, d_race_eth
+
+def get_test_series(chart_id: str, url: str) -> Tuple:
+	"""This method gets the date, the number of positive and negative tests on that date, and the number of cumulative positive and negative tests."""
+
+	csv_ = extract_csvs(chart_id, url)
+	series = []
+
+	csv_strs = csv_.splitlines()
+	keys = csv_strs[0].split(',')
+	
+	test_history = []
+	
+	# Grab the dates, which are in the header
+	for entry in csv_strs[:1][0].split(',')[1:]: 
```

Comment:
Could this just be:

```suggestion
	for entry in csv_strs[0].split(',')[1:]: 
```

But also, like above, I’d recommend using the `csv` module for parsing, so this should probably be more different anyway. ¯\\\_(ツ)\_/¯